docs/config.md

```diff
@@ -711,28 +711,29 @@ show-cryptographic-signatures = true
 
 ## Pager
 
-The default pager is can be set via `ui.pager` or the `PAGER` environment
-variable. The priority is as follows (environment variables are marked with
-a `$`):
+By default, jj will paginate output that would scroll off the screen. It does
+this by passing output through `less -FRX` on most platforms (on Windows it uses
+[the pager](#builtin-pager) that is built-in to jj). It is possible the default
+will change to `:builtin` for all platforms in the future.
```

Comment:
I'd delete the last sentence, since you haven't introduced the `ui.pager` variable yet, and this PR takes us in a different direction anyway. If we decide to, we can change the default to `:builtin` later either way.